tests/testthat/test-SingleCellExperiment.R

```diff
@@ -85,6 +85,41 @@ for (layer_key in names(ad$layers)) {
   })
 }
 
+test_that("to_SCE fails when providing duplicate assay names", {
+  expect_error(
+    ad$as_SingleCellExperiment(
+      x_mapping = "counts",
+      assays_mapping = c(counts = "numeric_matrix", integer = "integer_matrix")
+    ),
+    regexp = "duplicate names"
+  )
+})
+
+test_that("to_SCE works when only providing x_mapping", {
+  sce <- ad$as_SingleCellExperiment(x_mapping = "counts")
+  assay_names <- names(assays(sce))
```

Comment:
There is a function to get the names:

```suggestion
  assay_names <- assayNames(sce)
```